dev-python/xapp/xapp-2.0.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+DISTUTILS_USE_SETUPTOOLS=no
```

Comment:
```
 * Messages for package dev-python/xapp-2.0.1:

 * DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   value:    DISTUTILS_USE_SETUPTOOLS=no
 *   expected: DISTUTILS_USE_SETUPTOOLS=bdepend
```
It is the default value, so you should just remove this line.

Reply:
No problem. You added this line in January! ;-)